packages/engine-core/src/library/LibraryEngine.ts

```diff
@@ -93,6 +93,8 @@ export class LibraryEngine extends Engine {
     exitHooks.setListener(this, listener)
   }
 
+  // TODO: is there any example of someone calling this constructor without a custom library loader?
```

Comment:
Perhaps only in [`LibraryEngine.test.ts`](https://github.com/prisma/prisma/blob/e50510f05d7a4df6ee4b28e76753463fa3759347/packages/engine-core/src/__tests__/LibraryEngine.test.ts)